bundle/internal/bundletest/mutator_benchmark_test.go

```diff
@@ -0,0 +1,74 @@
+package bundletest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config/validate"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+	"github.com/stretchr/testify/assert"
+)
+
+func benchmarkRequiredMutator(b *testing.B, numJobs int) {
```

Comment:
Should it be a separate PR instead?

Reply:
I think the current PR is a better destination because the feature is co-versioned with its benchmarks. I don't mind either way though.